test/patch_test.dart

```diff
@@ -0,0 +1,41 @@
+// Copyright 2018 Google LLC
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import 'package:sass_module_migrator/src/patch.dart';
+import 'package:source_span/source_span.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test("single patch applied", () {
```

Comment:
Generally speaking, test names should be full sentences that succinctly describe the expected behavior. For example, "a single patch replaces the original text".